### DISCUSSION TITLE
Nuxt 3  build timeout : Build did not complete within 45 mins

### DISCUSSION THREAD

**Discussion by ChronicStone** [vercel/community#5593] [Help]
### Page to Investigate

https://exassess-cit27yj2w-exassess.vercel.app/

### Steps to Reproduce

Every build I run from my nuxt app stays hanging forever and then timeout. The thing is that the build process is actually done, 100% of it, after the build is complete the process just stays pending indefinitely.

https://exassess-cit27yj2w-exassess.vercel.app/

**Comment by pc7web** [comment]
I believe that nuxt, or any nuxt plugin, is the source of the issue rather than Vercel. Because I confirmed the same thing with Netlify as well.

**Comment by amyegan** [comment]
What does the project use for [Build and Development Settings](https://vercel.com/docs/deployments/configure-a-build#build-and-development-settings)?

**Reply by jose-madriz-wt** [reply]
Using default config here.
Build is reaching this :
[success] [nitro] You can preview this build using `node .output/server/index.mjs`
But fails to deploy.

**Comment by MTGPROD** [comment]
Same error as well since I updated my deployment, did you found any solutions since then ?

**Comment by amyegan** [comment]
I wasn't able to replicate the problem in my own test app. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use to help investigate the issue?
